### PR TITLE
Add a description for the karmadactl subcommand

### PR DESCRIPTION
**Pull request by Fish-pro**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

https://karmada.io/docs/next/reference/karmadactl/karmadactl-commands/karmadactl_index/
We can see that some of the subcommands are missing descriptions, and this pr modification is missing parts

**Which issue(s) this PR fixes**:
Part of https://github.com/karmada-io/karmada/issues/2665

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by lonelyCZ** [comment]
Thanks @Fish-pro 

/assign

File: pkg/karmadactl/describe.go [hunk 1]
```diff
@@ -15,6 +15,19 @@ import (
 )
 
 var (
+	describeLong = templates.LongDesc(`
+		Show details of a specific resource or group of resources in member cluster.
```
Comment (lonelyCZ) [inline pkg/karmadactl/describe.go:19]:
```suggestion
		Show details of a specific resource or group of resources in a member cluster.
```

File: pkg/karmadactl/logs.go [hunk 2]
```diff
@@ -17,8 +17,13 @@ const (
 )
 
 var (
+	logsLong = templates.LongDesc(`
+		Print the logs for a container in a pod in a cluster or specified resource. If the pod has
```
Comment (lonelyCZ) [inline pkg/karmadactl/logs.go:21]:
```suggestion
		Print the logs for a container in a pod or specified resource in a member cluster. If the pod has
```

**Comment by lonelyCZ** [comment]
/lgtm

Hi, @Fish-pro , please update your pr description `Fixes #2665` to `Part of #2665`

**Comment by Fish-pro** [comment]
> /lgtm
> 
> Hi, @Fish-pro , please update your pr description `Fixes #2665` to `Part of #2665`

@lonelyCZ Thanks, it has been updated

**Pull request merged**